### PR TITLE
H3 ignore unknown frames and uni streams

### PR DESCRIPTION
**Pull request by stammw**
Addresses the two failing tests on [quic tracker](https://quic-tracker.info.ucl.ac.be/grid).

### CODE REVIEW

File: quinn-h3/src/tests/helpers.rs [hunk 1]
```diff
@@ -47,6 +48,15 @@ pub struct Helper {
 
 impl Helper {
     pub fn new() -> Self {
+        if !TRACING_SUBSCRIBED.fetch_or(true, Ordering::Acquire) {
```
Comment (djc) [inline quinn-h3/src/tests/helpers.rs:51]: @Ralith can you review this commit?

File: quinn-h3/src/tests/helpers.rs [hunk 2]
```diff
@@ -1,12 +1,11 @@
 use std::{
     fs, io,
     net::{IpAddr, Ipv6Addr, SocketAddr},
-    sync::atomic::{AtomicU16, Ordering, AtomicBool},
+    sync::atomic::{AtomicBool, AtomicU16, Ordering},
```
Comment (djc) [inline quinn-h3/src/tests/helpers.rs:4]: This belongs in the previous commit. (Tip: `git rebase -x "cargo fmt -- --check"`.)

File: quinn-h3/src/tests/helpers.rs (hunk 1 above)
Comment (Ralith) [inline quinn-h3/src/tests/helpers.rs:51]: A simpler approach would be to call [`tracing_subscriber::fmt::try_init`](https://docs.rs/tracing-subscriber/0.2.5/tracing_subscriber/fmt/fn.try_init.html) and ignore its return value.

File: quinn-h3/src/tests/helpers.rs (hunk 2 above)
Comment (stammw) [reply quinn-h3/src/tests/helpers.rs]: Awesome!

File: quinn-h3/src/tests/helpers.rs (hunk 1 above)
Comment (stammw) [reply quinn-h3/src/tests/helpers.rs]: I was feeling this should be a thing, there it is. Thanks!

**Comment by stammw** [comment]
Got this deployed on the interop server. Let's wait to see if the tests go green before merging.

**Comment by Ralith** [comment]
Looks like the interop tracker had some sort of issue reaching our interop server over IPv4; strange. I don't see anything suspicious in the logs, save for a reboot this morning--maybe just really bad timing?

**Comment by stammw** [comment]
I was able to reproduce running quic-tracker on my machine. I'll investigate tomorrow, I've been busy lately.

**Comment by djc** [comment]
So can/should we merge this now?

**Comment by Ralith** [comment]
I'd still like to know why we're apparently dropping quic-tracker's packets on the floor all of a sudden.

**Pull request merged**